tag-input.md

```diff
@@ -8,33 +8,37 @@ Write a react component which will serve as an input for the spices used in cook
   - value: an array of strings representing the currently selected spices
   - onChange: a function which will fire any time 'value' changes
-- if the user presses 'enter' while typing:
-  - if there are any suggested spices, add the first one fo the list of selected spices
-    - otherwise, add a new selected spice whose value is the entire string that the user typed (allow users to add spices which are not on the list)
-  - clear out the typed text
+- If the user presses 'enter' while typing:
+  - If there are any suggested spices, add the first one fo the list of selected spices
+    - Otherwise, add a new selected spice whose value is the entire string that the user typed (allow users to add spices which are not on the list)
+  - Clear out the typed text
 
 ## Bonus points
 
 You may complete these steps if you find yourself with extra time.
 
-- if the user clicks on one of the suggestions, add it to the list of selected spices and clear out the typed text
-- if a user hits 'backspace':
-  - if the input is not selected, don't do anything
-  - if there is text that was typed by the user, backspace should behave normally (deleting any selected text, or the character immediately to the left of the cursor)
-  - if there is no text that was typed by the user, remove the most recently added selected spice from the list of selected spices
-  - if there are no selected spices, do nothing
+- If the user clicks on one of the suggestions, add it to the list of selected spices and clear out the typed text
+- If a user hits 'backspace':
+  - If the input is not selected, don't do anything
+  - If there is text that was typed by the user, backspace should behave normally (deleting any selected text, or the character immediately to the left of the cursor)
+  - If there is no text that was typed by the user, remove the most recently added selected spice from the list of selected spices
+  - If there are no selected spices, do nothing
 
 ## Hints
 
-1. Things do not have to look identical to the example image. What we care about the most is functionality, so focus on that before how the input looks.
-2. Write quality, readable code that you would feel comfortable with deploying to production. **If you are chosen to continue with the interview process, you will have to modify the code you write for the next step of the interview!**
-3. This type of component exists all over the internet, but we want to see you write your own. Third party libraries are OK to use, unless they completely trivialize the project. Use your best judgement, and reach out to your interviewer if you are unsure. We really just want to avoid having the entire project be completed with `npm install react-tag-autocomplete`.
+1. We recommend you prioritize in the following order:
+   - Finish base requirements
+   - Ensure high code quality
+   - Bonus points / style
+1. Focus on functionality before style - things do not need to look identical to the example image.
```

Comment:
1.
1.
1. 
?

Reply:
In markdown you can write a numeric list as
`1.`
`1.`
`1.`
`1.`
Etc and it gets translated to the real numbers on display. So you never have to update numbers if you add additional steps / remove steps etc

Reply:
see how it gets rendered:
https://github.com/shefhq/take-home-interview/blob/9d6fdb03ac53b69607eb351bf367192aef269aee/tag-input.md#hints

Reply:
see how it gets rendered:
https://github.com/shefhq/take-home-interview/blob/9d6fdb03ac53b69607eb351bf367192aef269aee/tag-input.md#hints